### ISSUE TITLE
Error running inference, Python version: 3.6.9

### ISSUE THREAD

**Issue by gokulanv** [closedloop-ai/cv19index#33]
Edit: Stack Trace below
**Error : UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 341: invalid start byte**
```
INFO:/content/cv19index/cv19index/predict.py:Reading model from /content/cv19index/cv19index/resources/xgboost_all_ages/model.pickle.  Writing results to examples/predictions.csv
DEBUG:cv19index.preprocess:Beginning claims data frame preprocessing, raw data frame as follows.
DEBUG:cv19index.preprocess:           personId  ... dx15
0  001ef63fe5cb0cc5  ...     
1  001ef63fe5cb0cc5  ...     
2  001ef63fe5cb0cc5  ...     
3  001ef63fe5cb0cc5  ...     
4  001ef63fe5cb0cc5  ...     

[5 rows x 21 columns]
DEBUG:cv19index.preprocess:Filtered claims to just those within the dates 2017-12-31 to 2018-12-31.  Claim count went from 68481 to 35735
DEBUG:cv19index.preprocess:Cleaning diagnosis codes.

A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
DEBUG:cv19index.preprocess:Computing diagnosis flags.
DEBUG:cv19index.preprocess:Preprocessing complete.
INFO:/content/cv19index/cv19index/predict.py:Reordering test inputs to match training.
INFO:/content/cv19index/cv19index/predict.py:Scale pos weight is 24.882477847848648. Rescaling predictions to probabilities
WARNING:/content/cv19index/cv19index/shap_top_factors.py:Computing SHAP scores.  Approximate = False
Setting feature_perturbation = "tree_path_dependent" because no background data was given.
Traceback (most recent call last):
  File "run_cv19index.py", line 6, in <module>
    cv19index.predict.main()
  File "/content/cv19index/cv19index/predict.py", line 454, in main
    do_run_claims(args.demographics_file, args.claims_file, args.output_file, args.model, args.as_of_date, args.feature_file)
  File "/content/cv19index/cv19index/predict.py", line 431, in do_run_claims
    predictions = run_xgb_model(input_df, model, quote = quote)
  File "/content/cv19index/cv19index/predict.py", line 376, in run_xgb_model
    **kwargs,
  File "/content/cv19index/cv19index/predict.py", line 161, in perform_predictions
    df_cutoff, model, outcome_column, mapping, **kwargs
  File "/content/cv19index/cv19index/shap_top_factors.py", line 127, in generate_shap_top_factors
    shap_values = shap.TreeExplainer(model).shap_values(
  File "/usr/local/lib/python3.6/dist-packages/shap/explainers/tree.py", line 121, in __init__
    self.model = TreeEnsemble(model, self.data, self.data_missing, model_output)
  File "/usr/local/lib/python3.6/dist-packages/shap/explainers/tree.py", line 726, in __init__
    xgb_loader = XGBTreeModelLoader(self.original_model)
  File "/usr/local/lib/python3.6/dist-packages/shap/explainers/tree.py", line 1326, in __init__
    self.name_obj = self.read_str(self.name_obj_len)
  File "/usr/local/lib/python3.6/dist-packages/shap/explainers/tree.py", line 1456, in read_str
    val = self.buf[self.pos:self.pos+size].decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 341: invalid start byte
---------------------------------------------------------------------------
CalledProcessError                        Traceback (most recent call last)
<ipython-input-9-4aabaf703f58> in <module>()
----> 1 get_ipython().run_cell_magic('shell', '', 'cd cv19index/\npython run_cv19index.py -a 2018-12-31 examples/demographics.csv examples/claims.csv examples/predictions.csv -f examples/features.csv\n# cv19index -a 2018-12-31 examples/demographics.csv examples/claims.csv examples/predictions.csv')

2 frames
/usr/local/lib/python3.6/dist-packages/google/colab/_system_commands.py in check_returncode(self)
    136     if self.returncode:
    137       raise subprocess.CalledProcessError(
--> 138           returncode=self.returncode, cmd=self.args, output=self.output)
    139 
    140   def _repr_pretty_(self, p, cycle):  # pylint:disable=unused-argument
```

**Comment by DaveDeCaprio** [comment]
This is apparently a problem with using XGBoost 1.1.0 and SHAP 0.35.  I think there are open tickets to fix.

The best solution right now is to downgrade xgboost to 1.0.2.
For more details and other workarounds, see https://github.com/slundberg/shap/issues/1215

**Comment by gokulanv** [comment]
Thank you, fixed this with  XGBoost 1.1.0 !
